src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -262,14 +278,15 @@ def test_create_with_set(self, local_client):
         assert run.tags == {"key": "val"}
         assert run.description == description
 
+    @pytest.mark.skipif(pf_recording_mode() == "replay", reason="Instable in replay mode.")
     def test_pf_flow_test(self):
         run_pf_command(
             "flow",
             "test",
             "--flow",
             f"{FLOWS_DIR}/web_classification",
             "--inputs",
-            "url=https://www.youtube.com/watch?v=o5ZQyXaAv1g",
+            "url=https://www.youtube.com/watch?v=o5ZQyXaAv1g&hl=de&persist_hl=1",
```

Comment:
may explain this magic (however, I know what it is) suffix somewhere, PR description or some where.